src/sql/src/plan/statement/ddl.rs

```diff
@@ -1217,6 +1214,11 @@ pub fn plan_view(
         finishing,
         depends_on,
     } = query::plan_root_query(scx, query.clone(), QueryLifetime::Static)?;
+
+    if let Some(dup) = desc.iter_names().filter_map(|c| c).duplicates().next() {
```

Comment:
micronit: this [could be flatten](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=16ae10e1fa0117217a83a0cb39fcbe27):

```suggestion
    if let Some(dup) = desc.iter_names().flatten().duplicates().first() {
```

Reply:
Ah, I knew I had derped this. ty!

Reply:
ah I didn't mean to include that `first`, it's not legal :-/

```rust
    if let Some(dup) = desc.iter_names().flatten().duplicates().next() {
```